### ISSUE TITLE
Tabs: Style tab

### ISSUE THREAD

**Issue by laurenmrice** [carbon-design-system/carbon-website#3827]
**Ref discussion:** [Original issue](https://github.com/carbon-design-system/carbon-website/issues/3669)
**Ref boxnote:** [Style tab](https://ibm.ent.box.com/file/1000662025583?s=dl1a1b7pzef6nuct3hqlimc1rqd7cfrj) PDF a11y comments

## Acceptance criteria

- Update the Tabs documentation on the Style tab on the website with a11y comments and using the new content template.

#### Color (enabled states)
- [ ] Consider adding the tabpanel container to the images and the styles to the structured lists.

#### Interactive states
- [ ] Need to clarify: Focus, is this an a11y violation with the selected state + focus?  There is a text weight change and a color change to make the text semibold instead of regular on the selected state and its text color is darker.
- [ ] Scrollable states: The chevron button does not get a focus state.
